cloudsql-ha-dr/README.md

```diff
@@ -1,30 +1,49 @@
 [![banner](../banner.png)](https://cloud.google.com/?utm_source=github&utm_medium=referral&utm_campaign=GCP&utm_content=packages_repository_banner)
 
 
-# Cloud SQL with High Availability and Disaster Recovery  
+# Host a Highly Available SQL database on Google Cloud
 
-## Description
+## Introduction
 
-This example demonstrates how to deploy a CloudSQL instance with high-availability and cross-region replica.
+This architecture uses click-to-deploy to demonstrate how to deploy a CloudSQL instance with high-availability and cross-region replica.
 
-Resources created:
-- VPC
-- CloudSQL for Postgres instance with [high-availability](https://cloud.google.com/sql/docs/postgres/high-availability)
-- CloudSQL [cross-region read replica](https://cloud.google.com/sql/docs/postgres/intro-to-cloud-sql-disaster-recovery) instance
+With data being a vital asset for organizations, the high availability and disaster recovery capabilities offered by Cloud SQL allow businesses to meet stringent data availability and regulatory requirements.
 
+By implementing this solution, organizations gain peace of mind knowing that their critical databases are protected and accessible, even during unforeseen events or outages. This solution ensures business continuity by minimizing downtime, providing robust data protection through automated backups, and enabling seamless failover and disaster recovery processes.
 
-:clock1: Estimated deployment time: 16 min 30 sec
+This solution is suitable for a wide range of applications and industries, including e-commerce platforms, financial systems, customer relationship management (CRM) tools, and more.
+
+## Use cases
+
+These are some examples of the use cases you can build on top of this architecture:
 
-:heavy_dollar_sign: Estimated solution cost: [USD 706.44 per 1 month](https://cloud.google.com/products/calculator/#id=358b4a68-4f82-4e88-84f4-1da99a05548b)
```

Comment:
I see you removed the estimation here but did not add it to the costs section. Could you please take a look?

Reply:
I actually moved this to the cost section under the architecture, the text changed but the link to the calculator is the same.